grr/server/grr_response_server/bin/grrafana.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python
+# Lint as: python3
+import os
+import json
+import typing
+from typing import Any, List, Tuple, Dict, Text
+
+from werkzeug.wrappers import Request, Response
+from werkzeug.wrappers.json import JSONMixin
+from werkzeug.routing import Map, Rule
+from werkzeug.exceptions import HTTPException, NotFound
+from werkzeug.middleware.shared_data import SharedDataMiddleware
+from werkzeug.utils import redirect
+from werkzeug.serving import run_simple
```

Comment:
Please sort the imports using the following principle:

1. First, all Python standard library imports in alphabetical order.
2. Second, all other imports, in alphabetical order. It's ok to group the import and separate groups by newlines, but the overall order has to be alphabetical.